src/app/control-planes/views/ControlPlaneDetailView.vue

```diff
@@ -108,15 +126,20 @@
 import { KUI_ICON_SIZE_30 } from '@kong/design-tokens'
 import { AddIcon } from '@kong/icons'
 
+import { GlobalInsightSource } from '../sources'
 import ErrorBlock from '@/app/common/ErrorBlock.vue'
+import LoadingBlock from '@/app/common/LoadingBlock.vue'
 import { useControlPlaneStatus } from '@/app/control-planes'
 import MeshInsightsList from '@/app/meshes/components/MeshInsightsList.vue'
 import type { MeshInsightCollectionSource } from '@/app/meshes/sources'
+import { useOnboardingAlert } from '@/app/onboarding'
 import ZoneControlPlanesList from '@/app/zones/components/ZoneControlPlanesList.vue'
 import type { ZoneOverviewCollectionSource } from '@/app/zones/sources'
 
 const ControlPlaneStatus = useControlPlaneStatus()
+const OnboardingAlert = useOnboardingAlert()
```

Comment:
Why don't we try decorating `ControlPlaneStatus` with `OnboardingAlert`?

That way there is zero code to do with Onboarding in the `ControlPlaneDetailView` (which is used for showing control-planes everywhere). Plus if you think about it, it actually works with "the status of the control plane is that it is a fresh install" so it belongs in `ControlPlaneStatus`. This is how we solve other instances of this problem.

That being said if you want to make the additional changes for this elsewhere first then I'd be fine approving this if all is ok and circling back at a later date to see if we can use the above approach instead. There is nothing technically here that wouldn't work, and the objective here is to move the onboarding notification to use `/global-insights`.total rather than `/meshes`.total which this PR does.

Lmk either way, but it would be good to get this PR resolved sooner rather than later.

Reply:
I’ve implemented this in https://github.com/kumahq/kuma-gui/pull/1746/commits/892b9c6de9d2e576e05d9b98772cf3ff8a3684f2 now.

Side note: Consumers of ControlPlaneStatus will also need to provide the props now required by this component.

Reply:
> Side note: Consumers of ControlPlaneStatus will also need to provide the props now required by this component.

Yeah thats kinda fine if we are changing the interface, all the props you've added here make sense for everywhere anyway. If not we could have also used DataSource and `can` inside the ControlPlaneStatus component which would avoid the interface change _and_ there would be a reason for diverging from the norm here. (Just incase: don't do this, just go with what you have already done and make the changes to the interface elsewhere where necessary)

Could you confirm that this works everywhere for me? There is multiple levels decoration going on for this (which is great and how it should work), but I haven't tested that personally. I kinda assume everything was fine?

Reply:
Yeah, I've checked it for two things decorating the component and saw both additions. I'll reference it in the appropriate place.

Reply:
Awesome 🎉  Sounds good then lets go with this. 1 sec lemme take one last look here